### DISCUSSION TITLE
Availability legacy

### DISCUSSION THREAD

**Discussion by lamerjack** [Koenkk/zigbee2mqtt#25292] [General]
So with 2.0 legacy availabilty gone forever?
I can reconfigure openhab but
it affect also the color of offline and online, before online was green now is grey...

**Comment by Koenkk** [comment]
The availability feature will not be removed with 2.0.0, only the way it is configured changes, see https://www.zigbee2mqtt.io/guide/configuration/device-availability.html on how to configure it.

**Comment by Wikibear** [comment]
But it's json now.
![grafik](https://github.com/user-attachments/assets/5c9347d1-0f59-4ce2-b8d7-6beda148a6c8)
